pkg/netpol/connlist/conns_formatter.go

```diff
@@ -104,16 +114,28 @@ func formExposureItemAsSingleConnFiled(peerStr string, exposureItem XgressExposu
 }
 
 // convertLabelsMapToString returns a string representation of the given labels map
+// considers the special labels (with requirements such as Exists, DoesNotExist, NotIn)
 func convertLabelsMapToString(labelsMap map[string]string) string {
-	return labels.SelectorFromSet(labels.Set(labelsMap)).String()
+	labelsSrings := make([]string, 0)
+	for k, v := range labelsMap {
+		if v == common.ExistsVal {
```

Comment:
can the `exists` keyword be imported from a pkg that handles selectors, instead of re-defining it in our `common` pkg?

Reply:
not relevant anymore